content/cpp/concepts/data-types/data-types.md

```diff
@@ -17,16 +17,29 @@ CatalogContent:
 
 C++ provides a rather large number of types. However, you can write perfectly good programs using only five of those:
 
-## Int
+## Memory Size
 
-`int` is a type for storing integer (whole) numbers. An integer typically requires 4 bytes of memory space and ranges from -2³¹ to 2³¹.
+As noted above, the data type of a variable also specifies the size of the information that it can hold:
+
+| Data Type     | Memory Size |
+| ------------- | ----------- |
+| `bool`        | 1 byte      |
+| `char`        | 1 byte      |
+| `int`         | 4 bytes     |
+| `float`       | 4 bytes     |
+| `double`      | 8 bytes     |
+| `std::string` | 24 bytes    |
+
+## int
```

Comment:
```suggestion
## Integers
```

For the most part, we prefer the headers be in "Title Case." What about a pluralization of the data type for a section? (e.g., "Integers")